    await client.connect()
    .then(res => {
        console.log("connect success:",res);
    })
    .catch(err => {
        console.error("connect error:",err);
    });
